pkgs/applications/office/notes/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, wrapQtAppsHook
+, qtbase
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "notes";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "nuttyartist";
+    repo = "notes";
+    fetchSubmodules = true;
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-pTRcXVwtH4gxxFIkJaqbuk6SOiC879oYxFgQyFBaQAc=";
+  };
+
+  cmakeFlags = [ "-DUPDATE_CHECKER=OFF" ];
+
+  nativeBuildInputs = [
+    cmake
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    qtbase
+  ];
+
+  meta = with lib; {
+    description = "A fast and beautiful note-taking app";
+    homepage = "https://www.get-notes.com";
+    downloadPage = "https://github.com/nuttyartist/notes";
+    license = licenses.mpl20;
+    platforms = platforms.linux;
```

Comment:
I see on their website that it also support Darwin, maybe we should reflect that in here?

Reply:
I don't know anything about Darwin, it would be better to submit a new PR by someone else.

Reply:
Maybe we could just enable the platform and see if it builds in here?

Reply:
Unexpected error: command failed with exit code 1

       > [ 69%] Building CXX object CMakeFiles/Notes.dir/src/treeviewlogic.cpp.o
       > [ 70%] Building CXX object CMakeFiles/Notes.dir/src/framelesswindow.mm.o
       > /tmp/nix-build-notes-2.2.0.drv-0/source/src/framelesswindow.mm:4:12: fatal error: 'Cocoa/Cocoa.h' file not found
       > #  include <Cocoa/Cocoa.h>
       >            ^~~~~~~~~~~~~~~
       > 1 error generated.
       > make[2]: *** [CMakeFiles/Notes.dir/build.make:975: CMakeFiles/Notes.dir/src/framelesswindow.mm.o] Error 1
       > make[1]: *** [CMakeFiles/Makefile2:98: CMakeFiles/Notes.dir/all] Error 2
       > make: *** [Makefile:136: all] Error 2
       For full logs, run 'nix log /nix/store/gs1k9wrjw21z240v8ngic7jaz0hw445g-notes-2.2.0.drv'.

Reply:
I think it's easy to fix, i'll check when I'm back in front of a laptop.

In the meantime, have you tried using `nix-init` ?

Reply:
Yeah, it only generate this:
```
  nativeBuildInputs = [
    cmake
  ];
```